### PR TITLE
Add information about the report filters to the report and add a link to reproduce the report.

### LINKED ISSUE #1631: `Download pdf` does not obey severity filter
**Describe the bug**
https://github.com/minvws/nl-kat-coordination/pull/469 introduced the "Download pdf" button and severity filters. However, when visiting the findings page with the severity filter, download pdf also includes other severities.

**To Reproduce**
1. Go to /en/<<organisation_id>>/findings/?severity=low
2. Click on 'Download pdf'
3. Notice other severities than `low` are included in the pdf

**Expected behavior**
`Download pdf` following the filters on the findings page, (or maybe a second button to either get all findings or only the filtered findings).

**OpenKAT version**
1.10.1

### PR DESCRIPTION
**Pull request by Donnype**
### Changes
Add information about the used filters to the report and add a link to use the same filters again.

### Issue link

Closes #1631 (again, kinda)

### Proof
Finding report:

image

Link: http://localhost:8000/nl/test/findings/?observed_at=2023-09-03&severity=recommendation&severity=medium

OOI report:

image

Link: http://localhost:8000/nl/test/objects/report/?observed_at=2023-09-05&depth=9&ooi_id=Network%7Cinternet

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [x] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by praseodym — APPROVED** [review]
lgtm

**Comment by Donnype** [comment]
@Darwinkel Voilà
Schermafbeelding 2023-09-13 om 11 33 49


And now the muted findings filters actually have effect:


Schermafbeelding 2023-09-13 om 11 47 15



Schermafbeelding 2023-09-13 om 11 47 48

**Comment by ammar92** [comment]
## Checklist for QA:
- [ ] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [ ] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
Seems to work:
image

**Review by ammar92 — APPROVED** [review]

File: keiko/templates/bevindingenrapport/model.py [hunk 1]
```diff
@@ -58,3 +58,4 @@ class DataShape(DataShapeBase):
     valid_time: datetime
     report_source_type: str
     report_source_value: str
+    filters: dict
```
[the remark this answers is not in the document]
Comment (Donnype) [reply keiko/templates/bevindingenrapport/model.py:61]: Well, there are no calls that do not pass a filter at this point and we probably will always have some observed_at/valid_time that was used as an argument for now. So no need for it to be optional for now?

**Comment by Donnype** [comment]
Now you can configure if you want the permalinks in the reports through the ROCKY_REPORT_PERMALINKS environment variable, which defaults to `True`.

**Review by ammar92 — APPROVED** [review]

**Review by dekkers — APPROVED** [review]

**Pull request merged**